src/leo-client-app/src/components/EditSchedules.css

```diff
@@ -0,0 +1,3 @@
+.h1{
+    color:rgb(233, 209, 209);
+}
```

Comment:
there is a styles sheet (styles.css) for which you can get all the colours and sizing from, use those instead of custom defining